service/collector/elb.go

```diff
@@ -195,9 +195,11 @@ func (e *ELB) collectForAccount(ch chan<- prometheus.Metric, awsClients clientaw
 			return microerror.Mask(err)
 		}
 
-		err = e.cache.Set(account, *elbInfo)
-		if err != nil {
-			return microerror.Mask(err)
+		if elbInfo != nil {
```

Comment:
Just for my interest, what implications does it have to not have the ELB info present? What does that mean for the collector or metrics we want to have?

Reply:
In case there is no ELB info for a given account we can not / do not emit metrics for it. We still emit metrics for the other accounts.

Reply:
That was the case before, too. The thing that changed is that now we skip the caching in ase there is no ELB info.

Reply:
Below seems to be the same condition again. Can we just put the logic together or should we keep it separated?

Reply:
I think we should keep it separated. We need to check if info is available before emitting metrics even if we didn't  try to cache.